src/gui/qgsowssourceselect.h

```diff
@@ -68,6 +69,30 @@ class GUI_EXPORT QgsOWSSourceSelect : public QgsAbstractDataSourceWidget, protec
 
     void reset() override;
 
+    /**
+     * Prepares the spatial extent box with the general settings
+     * including original crs, destination crs and the map
+     * canvas if it is available.
+     *
+     * \since QGIS 3.24
+     */
+    void prepareExtent() SIP_SKIP;
+
+    /**
+     * Sets the dialog map canvas
+     * \see mapCanvas()
+     * \since QGIS 3.24
+     */
+    void setMapCanvas( QgsMapCanvas *mapCanvas ) override;
```

Comment:
We usually avoid repeating the documentation for overridden methods, unless there's specific detail we need to add regarding the override:

```suggestion
    void setMapCanvas( QgsMapCanvas *mapCanvas ) override;
```